HISTORY.md

```diff
@@ -1,4 +1,8 @@
 # Rocksdb Change Log
+## Unreleased
+### Public API changes:
+* Remove ReadOptions::iter_start_seqnum which has been deprecated.
```

Comment:
What about preserve_deletes and SetPreserveDeletesSequenceNumber? I think they're part of the same feature. Will they be in a separate PR?